public/js/components/Editor.css

```diff
@@ -10,7 +10,7 @@
  */
 .editor-wrapper {
   position: absolute;
-  height: calc(100% - 30px);
+  height: calc(100% - 31px);
```

Comment:
Why 31? Wish we didn't have to hardcode these values... wasn't this just for Chrome? I remember our flexbox being better equipped at this.